src/Microsoft.Framework.Configuration.FileExtensions/FileConfigurationExtensions.cs

```diff
@@ -49,7 +49,13 @@ public static string GetBasePath(this IConfigurationBuilder configurationBuilder
                 return (string)basePath;
             }
 
-            return string.Empty;
+#if DOTNET
+            return AppContext.BaseDirectory;
+#else
+            var baseDir = AppDomain.CurrentDomain.GetData("APP_CONTEXT_BASE_DIRECTORY");
+
+            return (baseDir != null) ? (string)baseDir : AppDomain.CurrentDomain.BaseDirectory;
```

Comment:
@davidfowl can you take a look at this to make sure this is what you suggested we do?